shim/shim.c

```diff
@@ -296,25 +297,67 @@ handle_data(int infd, int outfd)
 	}
 }
 
+/*!
+ * Print help usage
+ *
+ * \param prog Binary name
+ */
+void print_usage(char *prog) {
+	printf("%s: Usage\n", prog);
+        printf("  -c,  --container_id   Container id\n");
+        printf("  -p,  --proxy_sock_fd  File descriptor of the socket connected to cc-proxy\n");
+        printf("  -h,  --help           Display this help message\n");
+}
+
 int
 main(int argc, char **argv)
 {
 	int proxy_sockfd;
 	struct sockaddr_un proxy_address;
-	char* container_id;
+	char* container_id = NULL;
 	char* proxy_sock_path;
 	struct pollfd poll_fds[MAX_POLL_FDS];
```

Comment:
I'd be tempted to set `poll_fds[MAX_POLL_FDS] = { -1 };` for symmetry with `proxy_sock_fd`.